README.md

```diff
@@ -80,7 +80,7 @@ require "resque_pause_helper"
 
 require "solid_queue"
 
-Resque.redis = Redis::Namespace.new "#{Rails.env}", redis: Redis.new(host: "localhost", port: 6379, thread_safe: true)
+Resque.redis = Redis::Namespace.new "#{Rails.env}", redis: Redis.new(host: "localhost", port: 6379)
```

Comment:
Is `thread_safe: true` the default on Redis 4 client?

Reply:
Superseded by https://github.com/rails/mission_control-jobs/pull/101